.github/workflows/build-test-docker.yaml

```diff
@@ -71,6 +79,8 @@ jobs:
         uses: docker/metadata-action@v4
         with:
           images: ${{ inputs.repository-name }}
+          flavor: |
```

Comment:
What is this flavor for? not sure I understand the use of this 'latest' tag

Reply:
The `flavor` parameter of the docker metadata action sets options that used for items in the `tags` section below. This makes it possible to add a common suffix to all tags in a single place rather than adding the suffix to each tag individually. The docs for this action are pretty good and explain the different configuration options in more detail if you're interested: https://github.com/docker/metadata-action#about